pkg/flags/flagset.go

```diff
@@ -57,6 +60,7 @@ func init() {
 	FlagSet.BoolVar(&ServerDryRun, "server-dry-run", true, "Enable/Disable server side dry run before updating resources")
 	FlagSet.DurationVar(&FetchTimeout, "fetch-timeout", 30*time.Second, "Timeout in seconds for fetching changes from repositories")
 	FlagSet.StringVar(&RepositoryDir, "repository-dir", "", "Directory in which to clone repositories. Defaults to cloning in memory if unset.")
+	FlagSet.Var(&GitTrack, "gittrack-mode", "Whether to manage GitTracks. Valid values are Disabled and Enabled")
```

Comment:
Is it possible to set defaults? We should have a default for `cluster-gittrack-mode` too preferably

Reply:
Is there any way to be more explicit? That's quite "magic" and kinda not obvious IMO. What's the default value for the other flag, `clustergittrack-mode`?